src/main/java/io/github/opencubicchunks/cubicchunks/core/asm/mixin/fixes/common/vanillaclient/MixinEntityPlayerMP.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  This file is part of Cubic Chunks Mod, licensed under the MIT License (MIT).
+ *
+ *  Copyright (c) 2015-2019 OpenCubicChunks
+ *  Copyright (c) 2015-2019 contributors
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package io.github.opencubicchunks.cubicchunks.core.asm.mixin.fixes.common.vanillaclient;
+
+import io.github.opencubicchunks.cubicchunks.core.server.vanillaproxy.IBedrockPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(EntityPlayerMP.class)
+public class MixinEntityPlayerMP implements IBedrockPlayer {
+    private boolean bedrock = false;
```

Comment:
No. Don't do mixins just to store a value you can easily store elsewhere. This information (which players are bedrock) should be trivial to store in VanillaNetworkHandler

Reply:
Also when done this way, does it persist after player respawn?

Reply:
I didn't test respawning actually... but I can't just put it in `VanillaNetworkHandler` because `VanillaNetworkHandler` is per-world, not global. I guess I could make it static...

Reply:
Should probably be static and cleared on player disconnect (there is a forge event for that).